### DISCUSSION TITLE
Custom blast databases

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#19] [General]
@gregdenay I pushed a first attempt at allowing users to provide their own blast databases. This required some substantial changes in foodme.nf . Seems to work tho - but we have no good way of validating that the database is "sane" beforehand, so incorrectly formatted blast databases will probably crash the pipeline with cryptic errors. 

While I was at it, I also changed `--gene` to `--database` since we no longer have only gene-level blast databases. 

If you find things dont work anymore, this is probably why

**Comment by gregdenay** [comment]
Ill check it next week, I still have an old version of Genbank somewhere. 

One could sanity check the database with:
```
blastdbcmd -db $DB -info
```
 For example:

```
if ! blastdbcmd -db $DB -info; then     
  echo error message
fi
```
